Xamarin.Forms.Platform.Android/FastRenderers/LabelRenderer.cs

```diff
@@ -198,6 +198,9 @@ protected virtual void OnElementChanged(ElementChangedEventArgs<Label> e)
 
 		protected virtual void OnElementPropertyChanged(object sender, PropertyChangedEventArgs e)
 		{
+			if (_disposed)
```

Comment:
a never found a solid way to repro it though :/ it was very random, Il see what i can wrap together next week